package.json

```diff
@@ -24,7 +24,7 @@
     "@emotion/css": "^11.1.3",
     "@grafana/aws-sdk": "0.3.0",
     "@grafana/data": "9.3.2",
-    "@grafana/e2e": "9.3.2",
+    "@grafana/e2e": "10.2.0",
```

Comment:
what's the reason for updating just the e2e package separately here? is it to do with the underscore package?

Reply:
Yes, first I tried to update @grafana/e2e to see if the underscore package dependency would be higher. It was not, but I did not revert the upgrade.
I didn't necessarily revert some other upgrades which didn't help. In this case I thought @grafana/e2e would be a harmless upgrade if it was only used in e2e tests.
Is this a case where you recommend that I leave it at its original version? Let me know!

Reply:
actually, it doesn't even look like there are any e2e tests and the e2e and e2e-selectors packages aren't even used in x-ray. we could just remove those deps as we'll eventually be moving to the updated e2e testing package later on.

Reply:
do you have some recommendation to check for dep usage? for example could i just find "grafana/e2e" by text in the files?

Reply:
Also I removed the e2e-selectors dependency and the scripts for e2e, if you could just confirm that is ok!

Reply:
@kevinwcyu I'm gonna go with this and merge, let me know if in hindsight it was not OK!